### PR TITLE
[VL] Support tencentos 3.2

### PR DESCRIPTION
**Pull request by yujiantao**
## What changes were proposed in this pull request?

Add support for tencentos 3.2 which is widely used in tencent 

## How was this patch tested?

manual test with tencentos 3.2

### CODE REVIEW

**Comment by zhanglistar** [comment]
Tencentos 3.2 is based on what?

**Comment by zhouyuan** [comment]
> Tencentos 3.2 is based on what?

it seems it's based on CentOS, with more kernel patches

`TencentOS Server 目前开放使用，用户态环境与 CentOS 保持兼容，在 CentOS 上开发的应用程序可直接在 TencentOS Server 上运行。`
https://cloud.tencent.com/document/product/1397/72777

File: backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/ContextInitializer.scala [hunk 1]
```diff
@@ -66,6 +66,8 @@ class ContextInitializer extends ContextApi {
       new SharedLibraryLoaderCentos8
     } else if (systemName.contains("Anolis") && systemVersion.startsWith("7")) {
       new SharedLibraryLoaderCentos7
+    } else if (system.contains("tencentos") && system.contains("3.2")) {
+      new SharedLibraryLoaderCentos8
```
Comment (PHILO-HE) [inline backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/ContextInitializer.scala:70]: Nit:
Please include the new supported OS in the below exception message (Line#74).

**Comment by PHILO-HE** [comment]
CI reported code format violation. Thanks!
```
error file=/opt/gluten/backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/ContextInitializer.scala message=File line length exceeds 100 characters line=74
```

**Comment by yujiantao** [comment]
> CI reported code format violation. Thanks!
> 
> ```
> error file=/opt/gluten/backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/ContextInitializer.scala message=File line length exceeds 100 characters line=74
> ```

**Comment by yujiantao** [comment]
> CI reported code format violation. Thanks!
> 
> ```
> error file=/opt/gluten/backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/ContextInitializer.scala message=File line length exceeds 100 characters line=74
> ```

fixed

**Review by PHILO-HE — APPROVED** [review]
Thanks for the contributing! Assume it has been well tested in your environment.

**Pull request merged**